stories/object-list/object-list.stories.js

```diff
@@ -599,6 +599,71 @@ export const selectionWithNavigation = () => `
 <h4 id="objectListItemRowSelectionAndNavigation">Object List Item With Row Selection And Navigation</h4>
 <div role="navigation" style="max-width: 450px">
 <ul class="fd-list fd-object-list fd-list--navigation-object fd-list--navigation fd-list--selection" role="listbox" aria-labelledby="objectListItemRowSelectionAndNavigation">
+<li role="option" id="check" style="outline:#fff;" aria-selected="true" tabindex="0" class="fd-list__item fd-object-list__item fd-list__item--link is-selected">
+<a tabindex="0" class="fd-list__link" style="background:#0854a0;outline-offset:-0.1875rem;outline:0.0625rem dotted #fff;"> 
+<div class="fd-object-list__container">
+      <div class="fd-object-list__intro" style="color:#fff;text-shadow: none;">
+       Optional inline text</span>
+      </div>
+      <div class="fd-object-list__header">
+        <span class="fd-avatar fd-avatar--s"
+         style="background-image: url('/assets/images/products/fitbit.png');color:#fff;text-shadow: none;">
+        </span>
+        <div class="fd-object-list__header-left">
+         <div class="fd-object-identifier fd-object-list__object-identifier">
+          <p class="fd-object-identifier__title" style="color:#fff;text-shadow: none;">
+           Fitbit Versa Smart Watch, Black Fitbit Versa Smart Watch
+          </p>
+         </div>
+        </div>
+        <div class="fd-object-list__header-right">
+          <span class="fd-object-number fd-object-list__object-number">
+            <span class="fd-object-number__text" style="color:#fff;">457.00</span>
+            <span class="fd-object-number__unit" style="color:#fff;">Euro</span>
+          </span>
```

Comment:
Hi @sKudum this is not what I meant. You are not testing anythings right now as you are hardcoding the colors for active state. We need to test how the colors for the background and text change when the item becomes "active" (is clicked/pressed). This is achieved with `is-active` class. 
In this case you show a selected state of the element by hardcoding `aria-selected="true"`:
<img width="493" alt="Screen Shot 2021-07-16 at 1 00 19 PM" src="https://user-images.githubusercontent.com/39598672/125983356-ab34fd83-74da-404b-b3ea-4f7a969a3d42.png">


You can achieve the same for active stat by adding `is-active` class to `fd-list__link`. You will have:
```
<a tabindex="0" class="fd-list__link is-active"> ....</a> 
```

This will lead to this view:
<img width="472" alt="Screen Shot 2021-07-16 at 1 04 06 PM" src="https://user-images.githubusercontent.com/39598672/125983714-d40c90bf-a26a-4075-9d35-08fa4ba97f69.png">


The background and the text color automatically change to dark blue and white because of the additional modifier class. This is how we test different states of the components. Like I said, ignore the focus line. If you insist on testing it you can always create a modifier class for focussed state or write JS code to focus the element all the time, but we don't do it usually. Inline styling is not an option as it won't work with theming and is basically useless.

Reply:
Hi @InnaAtanasova ,

 Thank You for the elaboration. I have removed the inline style and updated it as suggested. Please let me know if this is ok?

Thank You